ajax/libs/videojs-wavesurfer/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-wavesurfer",
+  "version": "1.3.2",
+  "description": "video.js plugin that adds a navigable waveform for audio and video files.",
+  "filename": "videojs.wavesurfer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/collab-project/videojs-wavesurfer.git"
+  },
+  "keywords": [
+    "waveform",
+    "audio",
+    "video",
+    "wavesurfer",
+    "videojs",
+    "videojs-plugin",
+    "player"
+  ],
+  "author": "Collab",
+  "license": "MIT",
+  "homepage": "http://videojs.com/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/collab-project/videojs-wavesurfer.git",
+    "fileMap": [
+      {
+        "basePath": "src/js",
+        "files": [
+          "**/*"
```

Comment:
Combined with my and hare1039's request. The rule would be `js/*wavesurfer*.+(js|map)`